app/router.js

```diff
@@ -76,6 +78,9 @@ module.exports = app => {
   })
   app.get(`/start`, startController)
 
+  app.get(`/area-selection`, areaSelectionController)
+  app.post('/area-selection', redirectToAssessmentList)
```

Comment:
Looking at the function `redirectToAssessmentList` the action there is really about persisting the user's selected region, maybe we could rename to something like `cacheSelectedUserRegion` 🤔

Reply:
it is redirecting to assessment list also, it is caching the region selected then redirecting to the Assessment list page.

`const redirectToAssessmentList = async (req, res) => {
  try {
    const { user } = req
    const areaInfo = JSON.parse(req.body.area)

    await cacheUserDetailsWithRegion(user.id, areaInfo.areaCode, areaInfo.areaName)

    return res.redirect(`/${devAssessmentId}/assessments`)
  } catch (error) {
    return res.render('app/error', { error })
  }
}`

Reply:
should we rename to cacheSelectedUserRegion anyway?

Reply:
I think it makes the intent clear - the redirection seems more a side affect of the action completing successfully